src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int maxRandomValue = 20;
+    private final int defaultRadius = 10;
```

Comment:
refresh knowledge about naming convention for constant
```suggestion
    private static final int MAX_RANDOM_VALUE = 20;
    private staticfinal int DEFAULT_RADIUS = 10;
```